libs/fairness/src/lib/v2/WizardReport.tsx

```diff
@@ -208,6 +207,8 @@ export class WizardReport extends React.PureComponent<IReportProps, IState> {
                 metrics={this.state.metrics}
                 fairnessPickerProps={this.props.fairnessPickerProps}
                 performancePickerProps={this.props.performancePickerProps}
+                chartKey={this.state.chartKey}
+                onUpdateChartKey={this.updateChartKey}
```

Comment:
seems it is unnecessary to keep another state here.
Can we simply pass the prop? instead of another copy of state?